src/glossary.rst

```diff
@@ -707,15 +713,58 @@ Glossary
       * :term:`run directory`
 
    source directory
-      Any directory where workflows are written and stored in preparation for
-      installation with ``cylc install`` or reinstallation with
-      ``cylc reinstall``.
+      Any directory where :term:`workflows <workflow>` are written and stored
+      in preparation for installation with ``cylc install`` or reinstallation
+      with ``cylc reinstall``.
 
       See also:
 
       * :term:`run directory`
       * :ref:`Installing-workflows`
 
+   run directory
+      This is a directory containing the configuration that Cylc uses to run
+      the :term:`workflow`.
+
+      Typically this is installed from the :term:`source directory` using
+      ``cylc install``.
+
+      The run directory can be accessed by a running workflow using
+      the environment variable ``CYLC_SUITE_RUN_DIR``.
+
+      See also:
+
+      * :term:`source directory`
+      * :term:`work directory`
+      * :term:`share directory`
+      * :term:`job log directory`
+      * :term:`cylc-run directory`
+
+   play
+      We run a :term:`workflow` using the ``cylc play`` command.
+
+      You can :term:`play`, :term:`pause` and :term:`stop` a :term:`flow`,
+      Cylc will always carry on where it left off.
+
+      See also:
+
+      * :term:`pause`
+      * :term:`stop`
+
+   pause
```

Comment:
just realised, there's already a definition for pause on master, is this branch a bit behind?